### ISSUE TITLE
ERROR  Cannot save a root without device[] or mountpoint[]

### ISSUE THREAD

**Issue by mortenmikkelsen** [jotta/jotta-cli-issues#126]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):** 
0.8.36055

**Description of problem:**
`jotta-cli add ~/data2/movies` results in
`ERROR  Cannot save a root without device[] or mountpoint[movies]`

**Expected:**
To be able to add the folder ~/data2/movies to backup. This has worked for an other hdd "data1" for a year, where ~/data1/movies is backed up. Installing a new hhd "data2", I can not add the folder ~/data2/movies to backup. Usergroup of both disks is jottad and permissions are the same.

**jotta-cli status (`jotta-cli status`):**
```
------------------------------------------------------------------------------
 Account   : xxxxx on Jottacloud
 Usage     : 5.60TiB / ( Unlimited )
 Device    : Linux-server
------------------------------------------------------------------------------
| Account is full. Go to https://www.jottacloud.com/web/account to resolve   | (still 400 GB free)
------------------------------------------------------------------------------
------------------------------------------------------------------------------
 Backups   :
------------------------------------------------------------------------------
   Path      : ~/data1/movies
   Files     : 389907 files / 5.16TiB
   Status    : 2 files (89.00KiB) have not been backed up

------------------------------------------------------------------------------
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
The log file does not contain info about this.

**Comment by Kimbsen** [comment]
Thanks for reporting the issue. It seems something has gone wrong patchingsome internal data from one version to the one your on now.

Could you send the db file from jottad's appdata to support@jotta.no. It would typically be at /var/lib/jottad/db .  It will help me understand what the current data looks like and why it could not patch properly.

**Comment by Kimbsen** [comment]
We've released version 0.8.36922 on unstable that should fix this issue.
